README.md

```diff
@@ -157,63 +264,115 @@ import { withRubberBand } from 'react-motions'
 const ComponentWithRubberBand = withRubberBand(<div>rubberBand!</div>)
 ```
 
-## withTada
+## Shake
+### Component
 
-Return a React Component with Tada animation (`1s` duration)
+Render a React Component with Shake animation (`2s` duration and iterationCount `infinite`)
 
 ```jsx
-import { withTada } from 'react-motions'
+import { Shake } from 'react-motions'
 
-const ComponentWithTada = withTada(<div>Tadaaaan!</div>)
+const ComponentShake = (
+  <Shake duration={2} infinite>
+    Let's shake here
+  </Shake>
+)
 ```
+### Function
 
-## withInfinite
+Return a React Component with Shake animation (`1s` duration)
 
-Set last animation with `infinity` property.
+```jsx
+import { withShake } from 'react-motions'
+
+const ComponentWithShake = withShake(<div>Let's shake here</div>)
+```
+## Swing
+### Component
+
+Render a React Component with Swing animation (`2s` duration and iterationCount `infinite`)
 
 ```jsx
-import { withInfinite, withShake } from 'react-motions'
+import { Swing } from 'react-motions'
 
-const DoNotStopBouncing = withInfinite(withShake(<div>Let's shake without stop!</div>))
+const ComponentSwing = (
+  <Swing duration={2} infinite>
+    Let's swing here
+  </Swing>
+)
 ```
+### Function
 
-## withSequence
+Return a React Component with Swing animation (`1s` duration)
 
-Execute next animation only after previous animation be finished. 
+```jsx
+import { withSwing } from 'react-motions'
+
+const ComponentWithSwing = withSwing(<div>Swing!</div>)
+```
+## Tada
+### Component
+
+Render a React Component with Tada animation (`2s` duration and iterationCount `infinite`)
 
 ```jsx
-import { withSequence, withShake, withJello } from 'react-motions'
+import { Tada } from 'react-motions'
 
-const SequencialAnimations = withSequence(
-  withShake,
-  withJello,
-  <div>First shake it then jello! </div>
+const ComponentTada = (
+  <Tada duration={2} infinite>
+    Let's tada here
+  </Tada>
 )
 ```
 
-## compose
+### Function
 
-Execute all animations in the same time.
+Return a React Component with Tada animation (`1s` duration)
 
 ```jsx
-import { compose, withFlash, withPulse } from 'react-motions'
+import { withTada } from 'react-motions'
 
-const VividAnimation = compose(
-  withFlash,
-  withPulse,
-  <div>Flash and Pulse!</div>
+const ComponentWithTada = withTada(<div>Tadaaaan!</div>)
+```
+## Wobble
+### Component
+
+Render a React Component with Wobble animation (`2s` duration and iterationCount `infinite`)
+
+```jsx
+import { Wobble } from 'react-motions'
+
+const ComponentWobble = (
+  <Wobble duration={2} infinite>
+    Let's wobble here
+  </Wobble>
 )
 ```
+### Function
+
+Return a React Component with Wobble animation (`1s` duration)
+
+```jsx
+import { withWobble } from 'react-motions'
+
+const ComponentWithWobble = withWobble(<div>Wobble!</div>)
+```
 
 # Roadmap
 
 - [ ] `withSequence`
 - [ ] `compose`
```

Comment:
Isn't both `withSquence` and `compose` done?

Reply:
nopes, He actually only worked on Components

Reply:
No, I did not do these, u can?

Reply:
You are just adding the documentation for how to use them, correct?

Reply:
Yes, I made a change in readme at the request of @raphamorim, though I did not created `withSquence` and `compose`.